eng/versioning/external_dependencies.txt

```diff
@@ -70,6 +71,7 @@ org.springframework.security:spring-security-oauth2-client;5.2.0.RELEASE
 org.springframework.security:spring-security-oauth2-core;5.2.0.RELEASE
 org.springframework.security:spring-security-oauth2-jose;5.2.0.RELEASE
 org.springframework:spring-web;5.2.5.RELEASE
+org.springframework:spring-jms;5.2.0.RELEASE
```

Comment:
This should be 5.2.5.RELEASE to match the org.springframework:spring-web's version. Both libraries have spring-beans and spring-core as immediate dependencies and unless these are the same version you risk introducing a dependency ordering issue.

Reply:
upgraded to 5.2.5.RELEASE